### PR TITLE
feat(react-dogfood): keyboard shortcuts

### PR DESCRIPTION
**Pull request by arnautov-anton**
- add toggle mute shortcuts for audio and video
- add raise hand reaction shortcut

### CODE REVIEW

**Comment by arnautov-anton** [comment]
I had to extract toggle functions from the actual buttons as they contain a ton of logic - was that what you meant before when you were writing the docs on CallControls, @MartinCupela?

File: packages/react-dogfood/hooks/useKeyboardShortcuts.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  useCall,
+  useToggleAudioMuteState,
+  useToggleVideoMuteState,
+} from '@stream-io/video-react-sdk';
+import { useEffect } from 'react';
+
+import hotkeys from 'hotkeys-js';
+
+enum Hotkeys {
+  TOGGLE_AUDIO_MAC = 'cmd+d',
```
Comment (oliverlaz) [inline packages/react-dogfood/hooks/useKeyboardShortcuts.ts:12]: Can we also have "cmd+space"? I think that is a more common shortcut that would be easier to remember.

Comment (arnautov-anton) [reply packages/react-dogfood/hooks/useKeyboardShortcuts.ts:12]: I used the ones used in the [Google Meet](https://support.google.com/a/users/answer/9896256?hl=en) - would you rather me to switch to the Slack preset?

Comment (oliverlaz) [reply packages/react-dogfood/hooks/useKeyboardShortcuts.ts:12]: can we have both? 😅

Comment (arnautov-anton) [reply packages/react-dogfood/hooks/useKeyboardShortcuts.ts:12]: image

**Review by oliverlaz — APPROVED** [review]

**Pull request merged**